unity-renderer/Assets/DCLPlugins/RealmsPlugin/RealmMinimapModifier.cs

```diff
@@ -0,0 +1,25 @@
+using DCL;
+using Decentraland.Bff;
+
+namespace DCLPlugins.RealmPlugin
+{
+    public class RealmMinimapModifier : IRealmModifier
```

Comment:
Same here - "What do you think about adding a short XML summary (just to main responsibility of the class)?"

Reply:
Done!